README.md

```diff
@@ -1,2 +1,24 @@
 # Hello Microverse test
- test for fix com
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+Hello Microverse
+
+This is my first project in the Microvers path, so i'm exciting and i want lo learn more about this path, let's go to learn more about codification.
+
+please put your feedback.
```

Comment:
- [ ] Good job so far by adding some changes required by the previous reviewer. So, I noticed that you didn't add all changes he told you. And he suggested you a good description of the project but you add the one which is not related to the project, it is like you describe another thing beyond the project you did.
Your description should describe what is about the project, what do you think if you read the one from [project's requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=In%20this%20project,to%20know%20them!) and make a short summary from it.

- [ ] As the previous reviewer said, it would be better to use the given README [template](https://github.com/microverseinc/readme-template). I suggest you first copy-paste the file and then you can edit it with related sections of your project, like adding: Authors sections to let know who wrote the project, and Issues section to help users comments on your project,...